src/test/java/uk/gov/pay/api/it/directdebit/MandatesResourceIT.java

```diff
@@ -102,12 +102,12 @@ public void createMandate(@Nullable String description) {
                 .body("_links.next_url_post.href", is(directDebitFrontendSecureUrl()))
                 .body("_links.next_url_post.method", is("POST"))
                 .body("_links.next_url_post.type", is("application/x-www-form-urlencoded"))
-                .body("_links.next_url_post.params.chargeTokenId", is(CHARGE_TOKEN_ID));
-                //TODO
+                .body("_links.next_url_post.params.chargeTokenId", is(CHARGE_TOKEN_ID))
+                .body("_links.payments.href", is("http://publicapi.url/v1/directdebit/payments?mandate_id=" + MANDATE_ID))
+                .body("_links.payments.method", is("GET"));
+                //TODO events url not defined yet
```

Comment:
I think we should add in the event link even though it doesn't exist yet. It will be `/directdebit/mandates/MANDATE_ID/events`

Reply:
can I do this in another PR? i'd like to leave this PR for the payment link.